centos_ci.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # bundle location
-BUNDLE_VERSION=4.6.1
+BUNDLE_VERSION=4.6.3
```

Comment:
Might be a good addition to the `check` target, error out if `grep BUNDLE_VERSION centos_ci.sh` is different from the version we have in the Makefile.